mke2fs.conf

```diff
@@ -2,46 +2,17 @@
 	base_features = sparse_super,large_file,filetype,resize_inode,dir_index
 	default_mntopts = acl,user_xattr
 	enable_periodic_fsck = 0
-	blocksize = 4096
-	inode_size = 256
-	inode_ratio = 16384
 
 [fs_types]
-	ext3 = {
-		features = has_journal
-	}
-	ext4 = {
+	# Based on ext4 with some features disabled
+	od-data = {
 		features = has_journal,extent
+		blocksize = 4096
 		inode_size = 256
```

Comment:
Do we gain anything from having 256-byte inodes instead of 128-byte? The man page mentions the possibility of storing extended attributes there, but I don't think anyone would be using that on a game system.

Reply:
I don't have any idea about this so I went with the default here

Reply:
Having 256-byte inodes allows [64-bit timestamps](https://ext4.wiki.kernel.org/index.php/Ext4_Disk_Layout#Inode_Timestamps) which both increases their resolution and wrap-around date. So that's probably worth having.